Doc/library/struct.rst

```diff
@@ -369,35 +401,130 @@ the result in a named tuple::
     >>> Student._make(unpack('<10sHHb', record))
     Student(name=b'raymond   ', serialnum=4658, school=264, gradelevel=8)
 
-The ordering of format characters may have an impact on size since the padding
-needed to satisfy alignment requirements is different::
-
-    >>> pack('ci', b'*', 0x12131415)
-    b'*\x00\x00\x00\x12\x13\x14\x15'
-    >>> pack('ic', 0x12131415, b'*')
-    b'\x12\x13\x14\x15*'
-    >>> calcsize('ci')
+The ordering of format characters may have an impact on size since the
```

Comment:
"since padding is implicit" in native mode? Still not entirely comfortable with wording, but this is what I have locally:

> The ordering of format characters may have an impact on size in native
> mode since padding is implicit. In standard mode, the user is
> responsible for inserting any desired padding.